x/supplier/keeper/msg_server_create_claim.go

```diff
@@ -25,7 +28,7 @@ func (k msgServer) CreateClaim(goCtx context.Context, msg *suppliertypes.MsgCrea
 		msg.GetSupplierAddress(),
 	)
 	if err != nil {
-		return nil, err
+		return nil, status.Error(codes.InvalidArgument, err.Error())
```

Comment:
Is this something we should be doing in message handlers, over returning an error from `types/errors.go`? Or is it specific for this use case that we are not doing so?

Reply:
(see: https://github.com/pokt-network/poktroll/pull/328#discussion_r1465982213)